src/diffusers/__init__.py

```diff
@@ -52,7 +52,7 @@
     from .utils.dummy_pt_objects import *  # noqa F403
 
 if is_torch_available() and is_scipy_available():
-    from .schedulers import LMSDiscreteScheduler
+    from .schedulers import EulerAncestralDiscreteScheduler, EulerDiscreteScheduler, LMSDiscreteScheduler
```

Comment:
Do these schedulers really require `scipy`? I didn't see that in the code. If they don't we should only check whether torch is available.

Reply:
Good catch Pedro!